repo-projects/vs-solutionpersistence.proj

```diff
@@ -0,0 +1,32 @@
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <ItemGroup>
+    <!-- Override default TargetFrameworks to skip building net framework. -->
+    <EnvironmentVariables Include="TargetFrameworks=$(NetCurrent)" />
```

Comment:
> It seems like the problem was that I specified net8.0 instead of $(NetCurrent). However, I also added a fix to change rollForward to disable. This should help prevent any strange behavior on build machines with multiple SDKs.

I didn't see your comment earlier.  I don't think this would be an issue given we control which SDK is used.

Reply:
I updated the PR to remove the rollForward update.